docs/user-guide/cli-using-initializing-team-configuration.md

```diff
@@ -48,3 +48,25 @@ The following steps describe how to interactively initialize team profile config
 
 **Important:** After the configuration files are in place (either via the zowe config init command or by manually creating the files), the now-deprecated zowe profiles commands will no longer function. Zowe CLI will return errors if you attempt to use deprecated profile commands.
 
+## Connecting profiles to API Mediation Layer
+
+To configure your profile to connect to API ML, issue the following command to set up your initial `zowe.config.json` file:
```

Comment:
If you are using the API Mediation Layer, you can automatically set up your `zowe.config.json` to access services registered to it by issuing the following command: